### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by beyzanurck**
creates API endpoints for CRUD operation,
creates WildlifeObservations components to displays data from 3 tables
creates individual, sightings, species modal to add new data

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,13 +1,132 @@
...
 
+  const [observation, setObservation] = useState([]);
+
+  const [show, setShow] = useState({
+    "species" : false,
+    "individuals" : false,
+    "sightings" : false
+  });
+
+  const handleClose = (type) => {
```
Comment (esolar07) [inline client/src/App.jsx]: @beyzanurck Not anything big... But you create one function for handlleClose and handleShow and just pass the type and true or false as parameter.

They use the same logic except for the type and bool in it. Try to always think DRY.

File: server/server.js [hunk 2]
```diff
@@ -1,14 +1,216 @@
...
+        res.status(500).send({ error: "Failed to fetch species." });
+    }
+});
+
+app.post('/species', async (req, res) => {
+    try {
+        const {common_name, scientific_name, estimated_number, conservation_code, created_at } = req.body;
+
+        const newSpecies = await db.query (
+            "INSERT INTO species (common_name, scientific_name, estimated_number, conservation_code, created_at) VALUES ($1, $2, $3, $4, $5) RETURNING *", [common_name, scientific_name, estimated_number, conservation_code, created_at]
```
Comment (esolar07) [inline server/server.js:28]: @beyzanurck Not sure if you all get this far but you should look at a ORM for DB queries. Makes life really ease with queries:

https://www.prisma.io/

But this still works as well.

**Comment by esolar07** [comment]
Nice work @beyzanurck. You did a great with the API routes. You kept them plural as well (which I do). You broke up the project in different components as well.

File: client/src/App.jsx (hunk 1 above)
Comment (beyzanurck) [reply client/src/App.jsx]: @esolar07 I combined them into the handleVisibility function. Thank you!

**Pull request merged**